<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EQBaseDatos</class>
 <widget class="QMainWindow" name="EQBaseDatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EQBaseDatos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/new/prefix1/Logo.png</normaloff>:/new/prefix1/Logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
	background-color: white;
    border: 1px solid dark;
    padding: 1px;
    border-radius: 3px;
    opacity: 200;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Totales</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Total Precio Compra: $0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Total Precio Venta: $0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Total Ganancia: $0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="Buscador">
      <property name="placeholderText">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>418</width>
        <height>297</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="Opcion">
      <item>
       <property name="text">
        <string>Producto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Buscar:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="AgregarFila"/>
   <addaction name="actionNuevo_Producto"/>
   <addaction name="EliminarProducto"/>
   <addaction name="separator"/>
   <addaction name="Buscar"/>
   <addaction name="separator"/>
   <addaction name="AdministrarUsuarios"/>
   <addaction name="LogOut"/>
   <addaction name="Salir"/>
  </widget>
  <action name="AgregarFila">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Icons/resources/iconos/Add Row-48.ico</normaloff>:/Icons/resources/iconos/Add Row-48.ico</iconset>
   </property>
   <property name="text">
    <string>Agregar Fila</string>
   </property>
   <property name="toolTip">
    <string>Agregar una fila vacia (Ctrl+A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionNuevo_Producto">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Icons/resources/iconos/Add-New-48.ico</normaloff>:/Icons/resources/iconos/Add-New-48.ico</iconset>
   </property>
   <property name="text">
    <string>Nuevo Producto</string>
   </property>
   <property name="toolTip">
    <string>Agregar un producto nuevo (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="EliminarProducto">
   <property name="icon">
    <iconset>
     <normalon>:/Icons/resources/iconos/Minus-48.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Eliminar Producto</string>
   </property>
   <property name="toolTip">
    <string>Eliminar los productos seleccionados (Ctrl+E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="Buscar">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Icons/resources/iconos/Search-48.ico</normaloff>:/Icons/resources/iconos/Search-48.ico</iconset>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="toolTip">
    <string>Buscar un producto por ID o por nombre (Ctrl+B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="AdministrarUsuarios">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Icons/resources/iconos/User-Modify-48.ico</normaloff>:/Icons/resources/iconos/User-Modify-48.ico</iconset>
   </property>
   <property name="text">
    <string>Administrar Usuarios</string>
   </property>
   <property name="toolTip">
    <string>Agregar o eliminar usuarios (Ctrl+U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="LogOut">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Icons/resources/iconos/Logout-48.ico</normaloff>:/Icons/resources/iconos/Logout-48.ico</iconset>
   </property>
   <property name="text">
    <string>Cerrar Sesión</string>
   </property>
   <property name="toolTip">
    <string>Cerrar Sesión (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Salir">
   <property name="icon">
    <iconset>
     <normalon>:/Icons/resources/iconos/iconmonstr-log-out-9-48.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Cerrar el programa</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
